jvm-packages/pom.xml

```diff
@@ -4,9 +4,9 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>ml.dmlc</groupId>
+    <groupId>ai.h2o</groupId>
     <artifactId>xgboost-jvm</artifactId>
-    <version>0.7</version>
+    <version>0.7.1</version>
```

Comment:
I thought you will introduce our version here .

Reply:
I'm doing it dynamically: 
```mvn -DnewVersion=${XGB_VERSION} \
            -Dxgboost.backend.type=${backend} \
            versions:set```